tests/unit_tests/dft/include/compute_inplace_real_real.hpp

```diff
@@ -29,8 +29,12 @@ int DFT_Test<precision, domain>::test_in_place_real_real_USM() {
     if (!init(MemoryAccessModel::usm)) {
         return test_skipped;
     }
-
-    try {
+    if constexpr (domain == oneapi::mkl::dft::domain::REAL) {
+        // storage schemes for real transform arn't allowed to set the REAL_STORAGE to
+        // real_real or the COMPLEX_STORAGE to real_real as well. Skip.
+        return test_skipped;
+    }
```

Comment:
I believe its more revalant if it is skipped here, as if real_real is supported in the future it will be obvious that this test would needed to be modified to handle the real case as well.

Reply:
+some messages stating test are skipped